ToDoList/src/services/api.ts

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+const api = axios.create({
+    baseURL: 'http://localhost:3000'
```

Comment:
I've hardcoded this url which will only work locally. Creating a story to create environment variables for apiUrl

Reply:
https://github.com/goldilocked/TodoListApplication/issues/12